packages/schema-extract/README.md

```diff
@@ -0,0 +1,217 @@
+# Schema Extract
+
+
+Schema extract transforms code files into [JSON-Schema](https://json-schema.org/) schemas. Schema extraction is a necessary step to create automated documentation for your code.
+
+It is composed of two parts, the **ts transformer** and **linker**.
+
+
+## Playground
+The `schema-extract` package has a playground that allows live editing of your code and transforms it to a schema and displays the results as you type it.
+
+## Usage
+
+Import the `createLinker` function from `@ui-autotools/schema-extract`. This function receives a string array of file paths and returns an initalized linker class.
+To get the linked schema of a specific file invoke `linker.flatten(fileName)`:
```

Comment:
`linker.flatten` is shown with one parameter here (fileName) but then used later with two parameters (fileName and interface?)

Reply:
Oh yeah, that's a mistake, there are 2 parameters